pipelines/configs/seqr.yml

```diff
@@ -1,12 +1,17 @@
 namespace: main
 analysis_dataset: seqr
-skip_datasets: ['ohmr3-mendelian', 'ohmr4-epilepsy', 'mito-disease', 'flinders-ophthal']
+#skip_datasets: ['ohmr3-mendelian', 'ohmr4-epilepsy', 'mito-disease', 'flinders-ophthal']
 skip_samples: [CPG11783,CPG12310,CPG199679,CPG13326]  # acute-care,acute-care,acute-care,perth-neuro
```

Comment:
Long term goal would be to move this to the SM somehow - I've added a card to our backlog to address this.

Reply:
Oh yeah. I shouldn't probably track this config under version control for now at all, just polluting commit contents.